web/src/message_view_header.ts

```diff
@@ -170,6 +170,54 @@ export function initialize(): void {
 export function render_title_area(): void {
     const filter = narrow_state.filter();
     build_message_view_header(filter);
+
+    $(".narrow_description").on("mouseenter mouseleave", function (event) {
+        const $description = $(this);
+        const windowWidth = $(window).width()!;
+        let hoverTimeout;
```

Comment:
We use snake case, not camel case, for JS variable names, so `window_width` and `hover_timeout` should be used here.